### ISSUE TITLE
Attribute AllureDisplayIgnored doesn't work with ignoring testcases

### ISSUE THREAD

**Issue by Ameterezu** [unickq/allure-nunit#24]
Hello!

If I use ignore attribute with testcases like this:
[TestCase("03/03/20166"), Ignore("PRX-1000")]
test doesn't exist in allure report.

**Comment by unickq** [comment]
Did you use `[AllureDisplayIgnored]` attribute?

**Comment by Ameterezu** [comment]
Sorry, I wrote not correct format of ignoring testcase.
This form doesn't work:
`[TestCase("test", Ignore = "test")]`

And this too:
```
private static IEnumerable SalesDocumentsSearchNotFoundCases
        {
            get
            {
                yield return new TestCaseData("Test")
                .SetName("{m}_NotExist")
                .Ignore("Test");
            }
        }
```

**Comment by unickq** [comment]
Hi @Ameterezu 

Did a small research

```csharp

[AllureDisplayIgnored]
[AllureNUnit]
internal class Class1
{
    [OneTimeSetUp]
    public void Clean()
    {
        AllureLifecycle.Instance.CleanupResultDirectory();
    }

    [Test]
    [Ignore("simple")]
    public void Simple()
    {
        Console.WriteLine("Simple");
    }

    [Test]
    public void AssertIgnore()
    {
        Console.WriteLine("Assert");
        Assert.Ignore("Assert");
    }

    [Test]
    [TestCase("a")]
    [TestCase("b", Ignore = "Case")]
    public void TestCaseIgnoreOne(string a)
    {
        Console.WriteLine(a);
    }

    [Test]
    [TestCaseSource(typeof(Class1), nameof(Data))]
    public void TestCaseData(string a)
    {
        Console.WriteLine(a);
    }


    private static IEnumerable Data
    {
        get
        {
            yield return new TestCaseData("Test").SetName("{m}_NotExist");
            yield return new TestCaseData("Ignore").SetName("{m}_NotExist").Ignore("Test");
        }
    }
}
```

Here's how ReSharper test runner sees it.
![image](https://user-images.githubusercontent.com/6696821/76762039-31b7ae80-6799-11ea-96e2-c9e582ab6ae0.png)

So the answer is yes - ignored cases are not included.
![image](https://user-images.githubusercontent.com/6696821/76762157-6a578800-6799-11ea-9359-09c8db3bc2bd.png)

**Comment by unickq** [comment]
I got all info for all runnable tests in that class like here https://github.com/unickq/allure-nunit/blob/e1971b8444a9ed2fe493df657d09403f3f042f30/src/allure-nunit/Attributes/AllureDisplayIgnoredAttribute.cs#L40
![image](https://user-images.githubusercontent.com/6696821/76763052-1352b280-679b-11ea-8b6a-2ce9630492b4.png)

There are only 4 - I can't get ignored test cases since they do not present in NUnit context

**Comment by Ameterezu** [comment]
Thank you

**Comment by Rumyash** [comment]
This is because the information about testcases is in the nested levels of ParameterizedMethodSuite.Tests. Did small fix for this #34
